admin/app/controllers/admin/commercial/AdsDotTextController.scala

```diff
@@ -0,0 +1,43 @@
+package controllers.admin.commercial
+
+import common.Logging
+import model.{ApplicationContext, NoCache}
+import play.api.data.Form
+import play.api.data.Forms._
+import play.api.i18n.I18nSupport
+import play.api.mvc.{Action, AnyContent, BaseController, ControllerComponents}
+import services.S3
+import conf.Configuration.commercial.adsTextObjectKey
+
+case class AdsTextSellers(sellers: String)
+
+object AdsTextSellers {
+    val form = Form(
+      mapping(
+        "sellers" -> nonEmptyText,
+      )(AdsTextSellers.apply)(AdsTextSellers.unapply))
+}
+
+class AdsDotTextController(val controllerComponents: ControllerComponents)(implicit context: ApplicationContext)
```

Comment:
Perhaps this should be `AdsDotTextEditController` and the other one `AdsDotTextViewController` for the sake of clarity? Or `EditAdsDotTextController` and `ViewAdsDotTextController`

Reply:
Good idea